docs/config.md

```diff
@@ -40,30 +57,28 @@ batch_sizes:
   step: 2
 ```
 
-- `<dict>`: a set of key-value pairs. Here is an example:
+- `<dict>`: a set of key-value pairs
 
 ```yaml
 triton_server_flags:
   log_verbose: True
   exit_timeout_secs: 120
 ```
 
-The above YAML declares the value of batch_sizes to be an array `[2, 4, 6]`.
-
-The following sections describe the configuration options for each of the
-subcommands of the Model Analyzer.
+<br>
 
-## Config options for `profile`
+# Config options for profile
 
-### CLI and YAML Config Options
+## CLI and YAML Config Options
 
-A list of all the configuration options supported by both the CLI and YAML
-config file are shown below. Brackets indicate that a parameter is optional. For
-non-list and non-object parameters the value is set to the specified default.
+A list of all the configuration options supported by **both the CLI and YAML**
+config file are shown below.
 
-The CLI flags corresponding to each of the options below are obtained by
-converting the `snake_case` options to `--kebab-case`. For example,
-`profile_models` in the YAML would be `--profile-models` in the CLI.
+- Brackets indicate that a parameter is optional.
+- For non-list and non-object parameters the value is set to the specified default.
+- The CLI flags corresponding to each of the options below are obtained by
+  converting the `snake_case` options to `--kebab-case`. <br>For example,
```

Comment:
I'm not sure why `<br>` is used in the documentation (versus a regular line break), but it seems like other spots that use it in the Model Analyzer repo break the line up. I think that'd be good here. In other words, breaking the line before `<br>` and having it on its own line. Or, at a minimum, at the end of a line (i.e. not followed by anything, for clarity).

Reply:
Fixed